shared-libs/transitions/src/transitions/index.js

```diff
@@ -238,7 +238,7 @@ const finalize = ({ change, results }, callback) => {
     // todo: how to handle a failed save? for now just
     // waiting until next change and try again.
     if (err) {
-      logger.error(`error saving changes on doc ${change.id} seq ${change.seq}: ${JSON.stringify(err)}`);
+      logger.error(`error saving changes on doc ${change.id} seq ${change.seq}: %o`, err);
```

Comment:
`console.log(JSON.stringify(new Error('you would expect something else, right?')));` => `{}`

Reply:
Ugh.